app/services/find_nearby.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+class FindNearby
+  AVAILABLE_PLACES = %w[car_dealer car_rental car_wash car_repair gas_station parking].freeze
```

Comment:
AVAILABLE_PLACES? Maybe AVAILABLE_SERVICES?

Reply:
Namespace is based on GOOGLE PLACES API: https://developers.google.com/places/web-service/search

Reply:
ok, AVAILABLE_TYPES based on : https://developers.google.com/places/web-service/supported_types